client/src/components/Recomendations/Recomendations.tsx

```
@@ -35,27 +37,45 @@ const Recomendations: React.FC = () => {
+                        <Marquee pauseOnHover vertical className="[--duration:20s]">
+                            {Array.from({ length: 6 }).map((_, index) => (
+                                <div key={index} className="flex items-center border border-gray-300 rounded-lg">
+                                    <div className="p-4 flex items-center space-x-4">
+                                        <div className="h-8 w-8 rounded-full bg-gray-300 animate-pulse"></div>
+                                        <div className="w-24 h-6 bg-gray-300 animate-pulse rounded-lg"></div>
+                                    </div>
+                                </div>
+                            ))}
+                        </Marquee>
+                    </ul>
+                </>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)